glide-core/README.md

```diff
@@ -1,5 +1,70 @@
 # Rust core
 
+## Prerequisites
+
+Please consider installing the following packages:
```

Comment:
Remove.  It's very polite, but not needed for a prerequisite section.  It makes it sound like the following packages are optional.